### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -56,23 +99,21 @@ func main() {
...
 			*serviceName,
 			*appPort,
-			*nativeContentAppAuth,
+			*sourceAppAuth,
 			*transformAppHostHeader,
-			*nativeContentAppUri,
+			*sourceAppUri,
 			*transformAppUri,
-			*nativeContentAppHealthUri,
+			*sourceAppAppHealthUri,
```
Comment (MihaiSorinMoisa) [inline app.go]: This variable has "App" twice in its name.

Comment (MMoisa) [inline app.go]: This variable has "App" twice in its name.

File: app.go [hunk 2]
```diff
@@ -103,43 +140,39 @@ func setupServiceHandler(sc ServiceConfig, metricsHandler Metrics, contentHandle
...
-	transformAppName          string
-	sourceAppPanicGuide       string
-	transformAppPanicGuide    string
-	graphiteTCPAddress        string
-	graphitePrefix            string
+	serviceName            string
+	appPort                string
+	sourceAppAuth          string
+	transformAppHostHeader string
+	sourceAppAppUri        string
```
Comment (MMoisa) [inline app.go]: This variable has "App" twice in its name.

**Review by MMoisa — CHANGES_REQUESTED** [review]

File: app.go (hunk 1 above)
Comment (kovacshuni) [reply app.go]: fixed

**Review by MMoisa — APPROVED** [review]

**Pull request merged**